oci/private/tarball.sh.tpl

```diff
@@ -32,4 +33,6 @@ config="blobs/${CONFIG_DIGEST}" \
 layers="${LAYERS}" \
 "${YQ}" eval \
         --null-input '.[0] = {"Config": env(config), "RepoTags": "${repotags}" | envsubst | split("%"), "Layers": env(layers) | map( "blobs/" + . + ".tar.gz") }' \
-        --output-format json > "${TARBALL_MANIFEST_PATH}"
+        --output-format json > "${STAGING_DIR}/manifest.json"
+
+tar -C "${STAGING_DIR}" -cf "${TARBALL_PATH}" manifest.json blobs
```

Comment:
add a TODO here that this is non-hermetic and maybe has portability problems, we'd like to get a proper tar toolchain from bazel-lib.